docs/DeveloperGuide.md

```diff
@@ -726,3 +880,60 @@ testers are expected to do more *exploratory* testing.
 
     1. Other incorrect find commands to try: `edit`, `edit x`, `...` (where x is the identification number which does not exist in the list)<br>
        Expected: Similar to previous.
+
+
+
+## **Appendix D: Effort**
+
+### Effort 
+
+#### New Addition to classes
+New classes were added which enhanced ClinicMate's Features. Below are some examples of the new classes added:
+
+`Model` classes: IdentityCardNumber, Sex, Note 
+
+`Command` classes: show, addnote
+
+We ensured that these new classes follows the code quality standard strictly and practice defensive programming.
+A couple of test cases were also crafted in order to ensure that the new classes function as expected and meet the specified requirements. These test cases cover various scenarios and edge cases to validate the robustness and reliability of the code. 
+
+#### Enhancements to existing functions
+
+Other than making changes to the existing commands by adding in more parameters, we put in significant effort to make sure the edited commands works to fit the features of ClinicMate.
+For instance, the `find` command will allow users to find contact details with their IC, and more importantly, the user's notes will also appear on the notes panel on te right side of the split pane.
+This allowed for a more comprehensive view of contact's notes which is an important field which our targeted users will require. Hence, there were much effort placed into enhancing existing features to improve usability of ClinicMate.
+
+
+#### Improve User Interface of ClinicMate 
+
+Rather than the dark theme in which ClinicMate originally had, we chose to go with a lighter theme and a bigger font size in order to improve readability of ClinicMate.
```

Comment:
'Rather than the dark theme which ClinicMate originally had,' --> removed the in